autoconfigure/collector-sqs/src/main/java/zipkin/autoconfigure/collector/sqs/ZipkinSQSCollectorAutoConfiguration.java

```diff
@@ -63,12 +63,12 @@ AWSCredentialsProvider credentialsProvider() {
    * doesn't have an option to treat empty properties as unset.
    *
    * <pre>{@code
-   * queueUrl: ${SQS_QUEUE_URL:}
+   * queue-url: ${SQS_QUEUE_URL:}
    * }</pre>
    */
   static final class SQSSetCondition extends SpringBootCondition {
 
-    private static final String PROPERTY_NAME = "zipkin.collector.sqs.queueUrl";
+    private static final String PROPERTY_NAME = "zipkin.collector.sqs.queue-url";
```

Comment:
The problem here is that the condition was matching on "queueUrl" instead of "queue-url".  When server config used "queue-url" the condition match fails and the SQS collector is not initialized.